### ISSUE TITLE
[Bug]:  Implicit conversion failed from VARCHAR to BIGINT UNSIGNED

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#3247]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

create table t1(bigint_col bigint);
select * from t1 where bigint_col='17666000000000000000';
Operator '=' with parameters [BIGINT UNSIGNED VARCHAR] will be implemented in future version.

**Comment by JinHai-CN** [comment]
Two cases also covered by #3247, another is fixed by #3672

[ERROR]
[SCRIPT   FILE]: ../matrixone_jh/test/cases/dtype/char.test
[ROW    NUMBER]: 10
[SQL STATEMENT]: SELECT CONCAT_WS(2,' test');
[EXPECT RESULT]:
CONCAT_WS(2,' test')
2 test
[ACTUAL RESULT]:
concat_ws(2,  test)
 test
 
 [ERROR]
[SCRIPT   FILE]: ../matrixone_jh/test/cases/dtype/char.test
[ROW    NUMBER]: 22
[SQL STATEMENT]: SELECT CONCAT_WS(14,'.3');
[EXPECT RESULT]:
CONCAT_WS(14,'.3') 
14.3
[ACTUAL RESULT]:
concat_ws(14, .3)
.3

**Comment by domingozhang** [comment]
fixed
